docs/examples/plot_complex_pipeline.py

```diff
@@ -100,36 +101,38 @@ def convert_dataframe_schema(df, drop=None):
             t = StringTensorType([1, 1])
         inputs.append((k, t))
     return inputs
-    
+
+
 inputs = convert_dataframe_schema(X_train)
 
 import pprint
 pprint.pprint(inputs)
 
 #############################
-# Merging single column into vectors is not 
+# Merging single column into vectors is not
 # the most efficient way to compute the prediction.
 # It could be done before converting the pipeline into a graph.
 
 ##################################
 # Convert the pipeline into ONNX
 # ++++++++++++++++++++++++++++++
 
-from skl2onnx import convert_sklearn
+from skl2onnx import to_onnx
```

Comment:
A safer way is to keep the convert_sklearn and add a deprecated warning.

Reply:
This is what I did. But I replaced convert_sklearn by to_onnx everywhere.

Reply:
This is what I did but it is better to use the version without a warning.